Packs/Base/Scripts/FindSimilarIncidentsByText/FindSimilarIncidentsByText.py

```diff
@@ -50,23 +50,26 @@ def get_incidents_by_time(incident_time, incident_type, incident_id, hours_time_
                                                                 incident_type)
 
     if ignore_closed:
-        query += " and -closed:*"
+        query += " and -status: closed"
 
     if incident_id:
         query += ' and -id:%s' % incident_id
 
     args = {'query': query, 'size': max_number_of_results, 'sort': '%s.desc' % time_field}
     if time_field == "created":
         args['from'] = min_date.isoformat()
-    res = demisto.executeCommand("getIncidents",
-                                 {'query': query,
-                                  'size': max_number_of_results, 'sort': '%s.desc' % time_field})
 
-    if res[0]['Type'] == entryTypes['error']:
-        raise Exception(str(res[0]['Contents']))
+    res = demisto.executeCommand('GetIncidentsByQuery', {
+        'query': query,
+        'fromDate': min_date.isoformat(),
+        'toDate': max_date.isoformat(),
```

Comment:
`isoformat` returns the time string in the following format:
![image](https://user-images.githubusercontent.com/50295826/120151902-81936180-c1f5-11eb-8804-d3920ea6b402.png)

Please let's make sure this format is good - if I recall correctly we had an issue with the millisecond info - @bakatzir am I right here?

Reply:
looks like the script we are using here is converting the date to this format anyway -
https://github.com/demisto/content/blob/master/Packs/Base/Scripts/GetIncidentsByQuery/GetIncidentsByQuery.py#L125-L140